R/strategies.R

```diff
@@ -1,16 +1,27 @@
+
+
+
 #' Compute covariance matrix for some reference-based methods (JR, CIR)
 #'
 #' @description
 #' Adapt covariance matrix in reference-based methods. Used for Copy Increments in
 #' Reference (CIR) and Jump To Reference (JTR) methods, to adapt the covariance matrix
-#' to different pre-deviation and post deviation covariance structures.
+#' to different pre-deviation and post deviation covariance structures. See Carpenter
+#' et al. (2013)
 #'
-#' @param sigma_group TODO
-#' @param sigma_ref TODO
-#' @param index_mar TODO
+#' @param sigma_group the covariance matrix with dimensions equal to `index_mar` for
+#' the subjects original group
+#' @param sigma_ref the covariance matrix with dimensions equal to `index_mar` for
+#' the subjects reference group
+#' @param index_mar A logical vector indicating which visits meet the MAR assumption
+#' for the subject. I.e. this identifies the observations that after a non-MAR
```

Comment:
I.e. this identifies the observations that after a non-MAR intercurrent event (ICE). -> I.e. this identifies the observations after a non-MAR intercurrent event (ICE).

Reply:
Done